client/src/components/HowToPlayComponent/HowToPlay.css

```diff
@@ -1,15 +1,18 @@
 .HowToplay {
   font-family: "Segoe UI", Roboto;
   font-size: 1rem;
-
+  background-color: #9b59b6;
```

Comment:
Please use color code `#35009D` for background and make it fill the entire background and not half.

Reply:
Sure will change it